### ISSUE TITLE
Onetastic for Mac OS

### ISSUE THREAD

**Issue by cloudmemory** [mjremijan/ferris-onetastic#1]
Are there any plans for a version of the macros for Mac OS?

Mac user, Onenote 2016

**Comment by mjremijan** [comment]
No, sorry, I do not have a Mac and I'm not planning on supporting a version Mac OS
